### DISCUSSION TITLE
Opening a second window slows a lot first window

### DISCUSSION THREAD

**Discussion by joaonunatings** [qutebrowser/qutebrowser#7195] [General] [answered]
Hi, recently I've had an issue where opening a second instance of qutebrowser slows down a lot the first one already open.
Im running qutebrowser 2.5.0-1 in sway 1.7

**Comment by The-Compiler** [comment, marked as answer]
Sounds somewhat similar to #4857.

**Reply by joaonunatings** [reply]
Setting _QT_QPA_PLATFORM=wayland_ fixed my issue.
